src/openrct2-ui/drawing/engines/opengl/OpenGLDrawingEngine.cpp

```diff
@@ -465,11 +465,11 @@ class OpenGLDrawingEngine final : public IDrawingEngine
             delete _smoothScaleFramebuffer;
             _smoothScaleFramebuffer = nullptr;
         }
-        if (GetContext()->GetUiContext()->GetScaleQuality() > 0)
+        if (GetContext()->GetUiContext()->GetScaleQuality() != ScaleQuality::NearestNeighbour)
         {
             _scaleFramebuffer = new OpenGLFramebuffer(_width, _height, false, false);
         }
-        if (GetContext()->GetUiContext()->GetScaleQuality() == SCALE_QUALITY_SMOOTH_NN)
+        if (GetContext()->GetUiContext()->GetScaleQuality() == ScaleQuality::SmoothNN)
```

Comment:
You can change that to a simple else statement here, both conditions can not apply at the same time.

Reply:
Not sure that's right, first if is true if ScaleQuality is either SmoothNN or Linear, 2nd is true only if ScaleQuality is SmoothNN. The variable they change is different each time too. Or maybe I'm not thinking straight if so sorry

Reply:
Actually my bad, you are correct.